### PR TITLE
Enable anonymous user statistics (opt-in)

### PR DESCRIPTION
**Pull request by ben-clayton**

### CODE REVIEW

**Comment by ben-clayton** [comment]
Will hold off submitting this until we get sign-off.
Created the PR so you can easily make any additional changes as needed.

**Comment by ben-clayton** [comment]
I've added a new CL that hooks up the crash reporting settings.

File: gapic/src/main/com/google/gapid/server/GapisProcess.java [hunk 1]
```diff
@@ -89,6 +89,11 @@ protected Exception prepare(ProcessBuilder pb) {
 
     String gapirFlags = "";
 
+    if (settings.reportCrashes) {
+      args.add("-crashreport");
+      args.add(settings.analyticsClientId);
```
Comment (pmuetschard) [inline gapic/src/main/com/google/gapid/server/GapisProcess.java]: -crashreport is a boolean flag, is it not?

Comment (ben-clayton) [reply gapic/src/main/com/google/gapid/server/GapisProcess.java]: Yes, good spot

**Pull request merged**